diff_testing/circ_gen.py

```diff
@@ -0,0 +1,63 @@
+"""To be test on fix/gate-errors on ssc branch."""
```

Comment:
I think this docstring can otherwise be reworded too, these branch names will not mean anything to the readers once the PRs are merged.
```suggestion
"""To be tested on fix/gate-error-compilation on ssc branch."""
```